deploy/olm-catalog/quay/1.0.2/quay.v1.0.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,370 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "redhatcop.redhat.io/v1alpha1",
+          "kind": "QuayEcosystem",
+          "metadata": {
+            "name": "example-quayecosystem"
+          },
+          "spec": {
+            "quay": {
+              "imagePullSecretName": "redhat-pull-secret"
+            }
+          }
+        },
+        {
+          "apiVersion": "redhatcop.redhat.io/v1alpha1",
+          "kind": "QuayEcosystem",
+          "metadata": {
+            "name": "example-quayecosystem"
+          },
+          "spec": {
+            "quay": {
+              "imagePullSecretName": "redhat-pull-secret",
+              "superuserCredentialsSecretName": "\u003csecret_name\u003e"
+            }
+          }
+        },
+        {
+          "apiVersion": "redhatcop.redhat.io/v1alpha1",
+          "kind": "QuayEcosystem",
+          "metadata": {
+            "name": "example-quayecosystem"
+          },
+          "spec": {
+            "clair": {
+              "enabled": true,
+              "imagePullSecretName": "redhat-pull-secret",
+              "updateInterval": "60m"
+            },
+            "quay": {
+              "configHostname": "example-quayecosystem-quay-enterprise.apps",
+              "configSecretName": "quay-config-app",
+              "database": {
+                "credentialsSecretName": "quay-database-credential",
+                "volumeSize": "10Gi"
+              },
+              "deploymentStrategy": "RollingUpdate",
+              "envVars": [
+                {
+                  "name": "DEBUGLOG",
```

Comment:
@sabre1041 It might be worth removing this environment variable.
The debug-level logging includes a *lot* of messages from Quay which aren't very important except for diagnostic purposes.

Reply:
These are examples that can be used. I reordered them so that the simple example should be the one that is shown by default in OLM